### ISSUE TITLE
Add Vlare support

### ISSUE THREAD

**Issue by spriteclad** [ytdl-org/youtube-dl#24245]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2020.03.01**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://vlare.tv/v/HSzfUoye
- Playlist: https://vlare.tv/u/xdb6VaaO/playlist/501

## Description

ytdl DOES support downloading videos (although, not playlists) by using the 'generic information extractor', but some built-in support for Vlare would be nice.

**Comment by savimanner** [comment]
There is no activity here for the last year, let's close this issue for now and re-open if needed.

**Comment by rautamiekka** [comment]
> There is no activity here for the last year, let's close this issue for now and re-open if needed.

No, you don't close tickets just cuz the ticket's been inactive.

**Comment by savimanner** [comment]
#24245#issuecomment-821228081

**Comment by savimanner** [comment]
> > There is no activity here for the last year, let's close this issue for now and re-open if needed.
> 
> No, you don't close tickets just cuz the ticket's been inactive.

vlare.tv is not up and running. Also the links provided by the `spriteclad` are not working.

**Comment by rautamiekka** [comment]
> > > There is no activity here for the last year, let's close this issue for now and re-open if needed.
> > 
> > 
> > No, you don't close tickets just cuz the ticket's been inactive.
> 
> vlare.tv is not up and running. Also the links provided by the `spriteclad` are not working.

True, you can't actually access the website, but
1) the website says it's coming back later the year.
2) https://twitter.com/VlareTV/status/1374760235693436937 says they're working on the site, that they're 25% ready. It was posted March 24.

, so the site's definitely not dead, but it's certainly inaccessible. Still, it's better than dead, which would warrant giving up on the ticket.
